db/seeds.rb

```diff
@@ -557,6 +557,28 @@
 
 ##############################################################################
 
+num_of_display_ads = 2 * SEEDS_MULTIPLIER
+
+seeder.create_if_none(DisplayAd) do
+  orgs_in_random_order = Organization.order(Arel.sql("RANDOM()"))
+  orgs = orgs_in_random_order.to_a
+
+  num_of_display_ads.times do
+    DisplayAd.create!(
+      organization: orgs.sample,
```

Comment:
For an initial state let's not associate an org id with the Billboard so that it will just show up on all posts. As a subsequent pass we can probably create some seeds with an org_id and others without it.

Reply:
Removing it.